zipkin/include/zipkin/zipkin_core_types.h

```diff
@@ -566,6 +569,7 @@ class Span : public ZipkinBase {
   std::string name_;
   uint64_t id_;
   Optional<TraceId> parent_id_;
+  bool sampled_;
```

Comment:
Could you default initialize sampled_ to true

Reply:
Definitely- just to check, do you mean via the default constructor of `Span` or do you mean in the definition of `sampled_`?

Reply:
in-class initialization is preferred for something like that: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c48-prefer-in-class-initializers-to-member-initializers-in-constructors-for-constant-initializers

A lot of the code was copied in from envoy, so not all of it is following some of the conventions I'd use.

Reply:
Thanks for the link- super helpful for me. I'll make the initialization change now.